cdap-distributions/src/emr/install.sh

```diff
@@ -24,9 +24,9 @@ CDAP_BRANCH=${CDAP_BRANCH:-release/5.0}
 # Optional tag to checkout - All released versions of this script should set this
 # like this: CDAP_TAG=${CDAP_TAG:+tag} as this allows setting tag to empty/null
 # otherwise, it should be CDAP_TAG=''
-CDAP_TAG=${CDAP_TAG:+v5.1.0}
+CDAP_TAG=${CDAP_TAG:+v5.1.2}
```

Comment:
should this be v6.0.0?

Reply:
We've been following the simple "set it to the next release" rule.  I assume develop will be version bumped to 5.1.3-SNAPSHOT?  In that case this should be 5.1.3.  Whether it happens now in this merge pr, or in a subsequent version bump PR doesn't matter (but i would suggest version bump PR, keeping this consistent with the rest of the branch).

Reply:
Develop is on 6.0.0-SNAPSHOT so I will put 6.0.0 here.

Reply:
nvm, looks like develop branch is already 6.0.0-SNAPSHOT so 6.0.0

Reply:
@dereklwood 

What should be the value above for 
```
CDAP_BRANCH=${CDAP_BRANCH:-release/5.0}
```
This is still at 5.0 should this be develop

Reply:
also update to next release:  `CDAP_BRANCH=${CDAP_BRANCH:-release/6.0}`.  

We should've updated 5.1/5.2 as well, but fortunately shouldn't have any impact as there have been no changes to [cdap-conf.json](https://github.com/cdapio/cdap/blob/684d78015ca5583aff368d54c4cf6a4e43ebc514/cdap-distributions/src/emr/cdap-conf.json)